pycaching/geocaching.py

```diff
@@ -494,8 +494,8 @@ def my_logs(self, log_type=None, limit=float("inf")):
                 break
 
             link = row.find(class_="ImageLink")["href"]
-            guid = parse_qs(urlparse(link).query)["guid"][0]
-            current_cache = self._try_getting_cache_from_guid(guid)
+            wp = link.split("/")[4]
```

Comment:
Could we make this more explicit somehow or check that the removed prefix is correct? For now, this seems to be "magic", as we do not have automated tests for this at the moment as well.

Reply:
An explicit way of doing this might be using regular expressions, which are not recommended as per contribution guidelines. Also note that on [line 470](https://github.com/tomasbedrich/pycaching/blob/15bc2acf2ba6e9141e850fb8296d8cf5058cce7f/pycaching/geocaching.py#L470), a very similar extraction is done in a very similar way.

Would you consider this resolved if a comment is provided similarly to line 470? Or would you rather see explicit checking of the prefix or moving the "logic" to a separate function?

Reply:
In the easiest way, we can consider adding a comment with an example URL here to make this more clear.

Reply:
I added the comment in commit cdeeeca, please see if this is sufficient.